cmd/sniproxy/sniproxy.go

```diff
@@ -174,6 +262,49 @@ func (s *server) serveConn(c net.Conn) {
 	p.Start()
 }
 
+// portNumberToName returns a human-readable name for several port numbers commonly forwarded,
+// and "tcp###" for everything else. It is used for metric label names.
+func (s *server) portNumberToName(forw *PortForward) string {
```

Comment:
This could be a plain top-level function, it doesn't use any state of `*server`.

Reply:
Done